### ISSUE TITLE
When I write in the settings file pbconf generate false.

### ISSUE THREAD

**Issue by damirsaifut** [ngx-grpc/ngx-grpc#98]
When I write in the settings file pbconf generate false. The following happens:
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/50329642/204885146-fc842fb8-c8ba-4771-9e4b-83d5311051a9.png">
This is a bug or it should be?

**Comment by smnbbrv** [comment]
Not really, the setting is only to generate the file itself or not. I'm not even sure / cannot recall why this setting is there. Probably it's just for being able to not generate the clients at all, when you need protobuf messages only

**Comment by damirsaifut** [comment]
> Not really, the setting is only to generate the file itself or not. I'm not even sure / cannot recall why this setting is there. Probably it's just for being able to not generate the clients at all, when you need protobuf messages only

Maybe I can fix it ?

**Comment by smnbbrv** [comment]
I would rather remove these options :)
